.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
+  "useTabs": false,
+  "arrowParens": "always",
+  "printWidth": 80,
+  "tabWidth": 2,
+  "singleQuote": true,
+  "trailingComma": "all",
+  "jsxBracketSameLine": false,
+  "semi": true
```

Comment:
I think `semi` should be set to false. I prefer to leave them out and it looks like the standard has been no semicolons in this project so far. @yknl @larrysalibra thoughts?

Reply:
Sure np, I prefer semis but nbd!